packages/website/src/pages/Search.tsx

```diff
@@ -44,13 +46,14 @@ const ResultListRenderer: FunctionComponent<ResultListProps> = (props) => {
                             {state.results.length === 1 ? ' result' : ' results'}
                         </p>
                         <div className="tile-grid">
-                            {state.results.map(({title, raw, uniqueId, clickUri}: Result) => (
+                            {state.results.map((result) => (
```

Comment:
Why did you remove the typing?

Reply:
Because I need the result object for the log :smile: 
https://docs.coveo.com/en/headless/0.28.1/reference/actions/click-analytics/#logdocumentopen